test/test_samplers.py

```diff
@@ -225,3 +227,23 @@ def simulate_one():
     sample = sampler.sample_until_n_accepted(10, simulate_one)
     assert 10 == len(sample.get_accepted_population())
     sampler.cleanup()
+
+
+def test_redis_catch_error():
+
+    def model(pars):
+        print(f"Model for {pars}")
+        if np.random.uniform() < 0.1:
+            raise ValueError("error")
+        return {'s0': pars['p0'] + 0.2 * np.random.uniform()}
+
+    def distance(s0, s1):
+        return abs(s0['s0'] - s1['s0'])
+    prior = Distribution(p0=RV("uniform", 0, 10))
+    sampler = RedisEvalParallelSampler(port=8775)
+    abc = ABCSMC(model, prior, distance, sampler=sampler)
+    db_file = "sqlite:///" + os.path.join(tempfile.gettempdir(), "test.db")
+    data = {'s0': 2.8}
+    abc.new(db_file, data)
+    with pytest.raises(Exception):
```

Comment:
I would assume that in the new implementation nothing gets raised, i.e. you can just remove the ``with pytest.raises`` and just call run

Reply:
When removing `with pytest.raises`, the test method will fail then since the model create error

Reply:
But the idea was that any errors are not propagated, but caught in the ``try-catch``, right?